api/src/controllers/team.js

```diff
@@ -310,6 +315,102 @@ async function updateTeam(req, res, next) {
   }
 }
 
+/**
+  * Clear the team Data from the database on /team/:team_id
+  * @param {} req request object, containing team id in the url
+  * @param {*} res response object, the deleted team document
+  * @returns 200: teamn is deleted
+  * @returns 404: team is not found
+  * @returns 400: team id is not in a valid hexadecimal format
+  */
+async function clearTeam(req, res, next) {
+  try {
+    const { team_id: _id } = req.params;
+    const { isDeleteFlag } = req.body;
+    await HomePage.deleteMany({ teamId: _id });
```

Comment:
After I click clear data, I got an error when refreshing the page. I think it's because the website document of that team had been deleted, but getWebsiteInfo api is called each time when the page is refreshed. 
If only the 'Clear Data' button is clicked (i.e. isDeleteFlag != true), I think the homepage and website shouldn't be deleted, instead, maybe just reset the properties inside these documents?